client/cypress/integration/visualizations/pivot_spec.js

```diff
@@ -19,6 +19,22 @@ const SQL = `
   SELECT 'c' AS stage1, 'c4' AS stage2, 44 AS value
 `;
 
+function createPivotThroughUI(visualizationName, options = {}) {
```

Comment:
I was going to use UI, but eventually realized that the API was needed. This function is a leftover from it and it's here because it seemed more organized than before.